src/tests/unit/common/blob-provider.test.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { provideBlob } from '../../../common/blob-provider';
+
+describe('BlobProviderTest', () => {
+    test('provideBlob', () => {
+        const mimeType = 'text/html';
+        const oMyBlob = provideBlob(['<a></a>'], mimeType);
```

Comment:
nit: This variable as well can have a more meaningful name.

Reply:
Ditto to my comment to Sebastian below--submitting for now to get into Canary, but will add in these suggestions in another PR! Thanks!